test_cli/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ if(BUILD_TESTING)
     PYTHON_EXECUTABLE "${_PYTHON_EXECUTABLE}"
     ENV
       INITIAL_PARAMS_RCLCPP=$<TARGET_FILE:initial_params_rclcpp>
+      INITIAL_PARAMS_RCLPY=test/initial_params.py
```

Comment:
I'm not sure what I would have expected the working directory to be when the test is run. If this passes I guess that means it is the top level source directory containing `CMakeLists.txt`.

Reply:
But I think it's also reasonable to set the source directory either via `CMAKE_CURRENT_LIST_DIR` or some other preferred variable for source paths.